### PR TITLE
Remove location safe from classes which are already inheriting

### PR DESCRIPTION
**Pull request by Rohit25negi**
Added test to ensure by default all views on dashboard are location safe.

@calellowitz  @emord

### CODE REVIEW

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -1653,8 +1627,8 @@ def post(self, request, *args, **kwargs):
         return redirect(self.urlname, domain=self.domain)
 
 
+@location_safe
```
Comment (emord) [inline custom/icds_reports/views.py]: This wasn't location_safe before. Why does it need to be now?

**Review by emord — APPROVED** [review]
just one question. will let @calellowitz do another pass.

**Review by calellowitz — APPROVED** [review]
Pending emord's question seems fine

File: custom/icds_reports/views.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/views.py]: Intent is to  allow the Bug report view to location restrict users As well. M I missing something here?
@emord

Comment (emord) [reply custom/icds_reports/views.py]: So its expected that this is a change in behavior? that was not obvious from this PR

Comment (Rohit25negi) [reply custom/icds_reports/views.py]: Like it was not requested by someone else. I changed with my understanding of Bug report view. Should not it be by default nature.

Comment (Rohit25negi) [reply custom/icds_reports/views.py]: removing this

**Pull request merged**